gnd/src/main/java/com/google/android/gnd/persistence/geojson/GeoJsonExtent.java

```diff
@@ -48,10 +48,10 @@
       return ImmutableList.of();
     }
 
-    double south = sw.optDouble(0, 0.0);
-    double west = sw.optDouble(1, 0.0);
-    double north = ne.optDouble(0, 0.0);
-    double east = ne.optDouble(1, 0.0);
+    double south = sw.optDouble(1, 0.0);
```

Comment:
This code will work, but the variable names may not always be correct because we can't count on the polygon having the sw coordinate first. Why not just tread the polygon vertices as an arbitrary list of coordinates?

Reply:
fantastic point! I've updated the implementation to better reflect this 7b660a7